_includes/release-version.html

```diff
@@ -109,49 +109,24 @@ <h5 class="version-date">
           <a class="btn btn-default dropdown-toggle" data-toggle="dropdown">
             Downloads&nbsp;&nbsp;&nbsp; <span class="caret"></span>
           </a>
+
           <ul class="dropdown-menu">
-            <li>
-              <a
-                href="https://repo1.maven.org/maven2/io/debezium/debezium-connector-mysql/{{releaseDetails[subVersion].version}}/debezium-connector-mysql-{{releaseDetails[subVersion].version}}-plugin.tar.gz"
-                >MySQL Connector Plug-in</a
-              >
-            </li>
-            <li>
-              <a
-                href="https://repo1.maven.org/maven2/io/debezium/debezium-connector-mongodb/{{releaseDetails[subVersion].version}}/debezium-connector-mongodb-{{releaseDetails[subVersion].version}}-plugin.tar.gz"
-                >MongoDB Connector Plug-in</a
-              >
-            </li>
-            <li>
-              <a
-                href="https://repo1.maven.org/maven2/io/debezium/debezium-connector-postgres/{{releaseDetails[subVersion].version}}/debezium-connector-postgres-{{releaseDetails[subVersion].version}}-plugin.tar.gz"
-                >PostgreSQL Connector Plug-in</a
-              >
-            </li>
-            <li>
-              <a
-                href="https://repo1.maven.org/maven2/io/debezium/debezium-connector-oracle/{{releaseDetails[subVersion].version}}/debezium-connector-oracle-{{releaseDetails[subVersion].version}}-plugin.tar.gz"
-                >Oracle Connector Plug-in</a
-              >
-            </li>
-            <li>
-              <a
-                href="https://repo1.maven.org/maven2/io/debezium/debezium-connector-sqlserver/{{releaseDetails[subVersion].version}}/debezium-connector-sqlserver-{{releaseDetails[subVersion].version}}-plugin.tar.gz"
-                >SQL Server Connector Plug-in</a
-              >
-            </li>
-            <li>
-              <a
-                href="https://repo1.maven.org/maven2/io/debezium/debezium-connector-cassandra/{{releaseDetails[subVersion].version}}/debezium-connector-cassandra-{{releaseDetails[subVersion].version}}-plugin.tar.gz"
-                >Cassandra Plug-in</a
-              >
-            </li>
-            <li>
-              <a
-                href="https://repo1.maven.org/maven2/io/debezium/debezium-connector-db2/{{releaseDetails[subVersion].version}}/debezium-connector-db2-{{releaseDetails[subVersion].version}}-plugin.tar.gz"
-                >Db2 Plug-in</a
-              >
-            </li>
+            {% for connector in releaseDetails.series.connectors %}
+              {% if connector.id == 'postgresql' %}
```

Comment:
Can't we use "postgres" as the id then to avoid this special handling?

Reply:
Yes, We can use `postgres` as it's not being used anywhere else apart from release pages.